src/SFA.DAS.Roatp.Functions/ApplicationExtract.cs

```diff
@@ -224,10 +228,55 @@ public async Task<List<SubmittedApplicationAnswer>> ExtractAnswersForApplication
                     }
                 }
             }
+            //Extract QuestionId Y0-110 answer even if it is inactive, there is an issue setting answer active to false while adding orgnisation type answer to the tabular list as the last entry
+            var hasPartnershipAnswersExtracted = answers.Any(answer => answer.QuestionId == QuestionIdPartnership);
+            if(!hasPartnershipAnswersExtracted)
+            {
+                await ExtractPartnershipAnswers(applicationId, answers);
+            }
 
             return answers;
         }
 
+        private async Task ExtractPartnershipAnswers(Guid applicationId, List<SubmittedApplicationAnswer> answers)
+        {
+            try
+            {
+                var partnershipAnswers = await ExtractAnswersByQuestionTag(applicationId, AddPartners, QuestionIdPartnership);
+                if (partnershipAnswers!= null)
+                {
+                    var tabularData = JsonConvert.DeserializeObject<TabularData>(partnershipAnswers.Value);
+
+                    var question = new Question
+                    {
+                        QuestionId = QuestionIdPartnership,
+                        Input = new Input
+                        {
+                            Type = TabularDataType
+                        }
+                    };
+                    var partnershipTabularAnswers = TabularDataMapper.GetAnswers(applicationId, YourOrganisation, WhosInControl, PageIdPartnershipAddPartners, question, tabularData);
+                    answers.AddRange(partnershipTabularAnswers);
+                }
+            }
+            catch (Exception ex)
+            {
+                _logger.LogError(ex, $"Unable to extracted Partnership answers for Application: {applicationId}");
+            }
+        }
+
+        private async Task<Answer> ExtractAnswersByQuestionTag(Guid applicationId, string questionTag, string questionId = null)
```

Comment:
string questionId = null shouldn't be the default, please take off the = null

Reply:
Done